app/assets/stylesheets/responsive/alaveteli_pro/_requests_style.scss

```diff
@@ -126,7 +126,8 @@ $complete: rgb(112, 203, 99);
 }
 
 .new-request-button,
-.new-request-button:visited {
+.new-request-button:visited,
+.new-request-button:hover {
```

Comment:
I think these styles aren't being used at all anymore as the class of the button been changed to just `button`. Can these be removed?

Reply:
It appears so - will remove